src/hooks/api/functions/serialize.ts

```
@@ -46,21 +54,20 @@ export async function serialize(client: Client, messages: WAMessage) {
+    prefix: prefix,
     sender: sender,
-    text: extractTextFromMessage(message!),
+    text: extractStringfromMessage(message!),
     mentions: Quoted ? Quoted.mentionedJid : [],
     mode,
-    sudo: sudo.includes(sender!) || sender === owner,
-    loadMessage,
+    sudo: (await getSudo())?.includes(sender!) || sender === owner,
     user: function (match?: string): string | undefined {
       if (this.isGroup) {
         if (this.quoted && this.quoted.sender) return this.quoted.sender;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)